docs/training.md

```diff
@@ -0,0 +1,4 @@
+## Introduciton
+
+
```

Comment:
Go to the "Files changed" tab and add a line comment to the Pull Request mentioning that you should add your favorite color.